libs/accounts/recovery-phone/src/lib/recovery-phone.errors.ts

```diff
@@ -42,3 +43,18 @@ export class RecoveryNumberNotSupportedError extends RecoveryPhoneError {
     super('Phone number not supported.', { phoneNumber }, cause);
   }
 }
+
+export class SmsSendRateLimitExceededError extends RecoveryPhoneError {
+  constructor(
+    uid: string,
+    toPhoneNumber: string,
+    fromPhoneNumber: string,
+    cause?: Error
+  ) {
+    super(
+      'Too many SMS are currently being sent. Try again later.',
```

Comment:
If this is meant to be user-facing do we want to add this in `auth-errors` and localize?

Reply:
We have this: https://mozilla-hub.atlassian.net/browse/FXA-10977
I'll add the new error type to the list there

Reply:
Thanks @vpomerleau